### ISSUE TITLE
`no-global-object-property-assignment` doesn't trigger for logical assignment operators

### ISSUE THREAD

**Issue by FloEdelmann** [sindresorhus/eslint-plugin-unicorn#3471]
```js
window.foo = {}   // correctly reports
window.foo ||= {} // does not report
```

**Comment by FloEdelmann** [comment]
Nevermind, it's already reported correctly.
